sigs.yaml

```diff
@@ -49,7 +49,7 @@ sigs:
   - name: Architecture
     dir: sig-architecture
     mission_statement: >
-      The Architecture SIG maintains and evolves the design principles of Kubernetes, and provides a consistent 
+      The Architecture SIG maintains and evolves the design principles of Kubernetes, and provides a consistent
```

Comment:
Yeah, I just split it out into https://github.com/kubernetes/community/pull/849 so that it's not tied to this (it's unrelated) and can be merged separate from the review of this change.